website/client/src/components/tasks/spells.vue

```diff
@@ -300,7 +300,7 @@ export default {
       const { user } = this;
       if (!user.party.quest) return 0;
 
-      const userQuest = this.quests[user.party.quest.key];
+      const userQuest = this.quests.quests[user.party.quest.key];
```

Comment:
hmm I thought this was an error but looks like we've been using the wrong object for a long time... good catch!